include/pybind11/pytypes.h

```diff
@@ -105,6 +116,7 @@ class object_api : public pyobject_tag {
         function will throw a `cast_error` exception. When the Python function
         call fails, a `error_already_set` exception is thrown.
     \endrst */
+    // clang-format on
```

Comment:
Wouldn't disabling reflow_comments be better? Or there may be some of there might be some way to get it to recognize \rst \endrst as off and on clang-format comments.